examples/README.md

```diff
@@ -0,0 +1,64 @@
+Creating a basic S2I builder image  
+---
+
+### Getting started  
+
+Directory skeleton:  
+
+* `Dockerfile` – standard Dockerfile where we’ll define the base builder image.  
+* `assemble` - responsible for building the application.  
+* `run` - responsible for running the application.  
+* `usage` - optional script responsible for printing the usage of the builder.  
+* `save-artifacts` - optional script for incremental builds that save built artifacts.  
+
+We are going to start by adding our logic to the [`Dockerfile`](nginx-app/Dockerfile).  
+
+In our [`assemble`](nginx-app/assemble) script we define other actions that need to be run when we want to install our application, this is where we build python modules or bundle install our required components, however in this example we leave it as it is. It will only override the default index.html file. It will also copy over any previously saved artifacts.  
+
+Let's create our [`run`](nginx-app/run) script that will start the application.  
+
+We can also specify a logic for an incremental build in [`save-artifacts`](nginx-app/save-artifacts).    
+
+We are now ready to create our builder image that contains nginx and we will name it `nginx-centos7`.
+
+`docker build -t nginx-centos7 .`
+
+With the builder image created we can now create the application image (the builder image plus a sample source repository).
+More [here](https://github.com/openshift/source-to-image/blob/master/docs/builder_image.md).
+
+```
+s2i build test/ nginx-centos7 nginx-app
+---> Building and installing application from source...
+```
+We can also print out the [`usage`](nginx-app/usage) format of the image.
```

Comment:
you should have them run this against the nginx-centos7 image, not the nginx-app image.

put this step before the "s2i build" invocation.  that is, the flow should be:

1) docker build the builder image
2) illustrate the usage being printed, which also tells you how to invoke s2i build with it
3) have them invoke s2i build with it (which, coincidentally, is what the usage just told them to do)
4) have them run the resulting application